### PR TITLE
Fix Spelling Errors

### PR DESCRIPTION
**Pull request by nicklambourne**
Goddammit Jimothy, see what your negligence hath wrought!

### CODE REVIEW

File: committee_history.md [hunk 1]
```diff
@@ -117,7 +117,7 @@
   - Tom Cranitch
   - Darren Fu
   - Madhav Kumar
-  - Matt Low
+  - Matthew Low
   - Olivia MacKenzie-Ross
```
Comment (mattchrlw) [inline committee_history.md:121]: For the sake of pedantry, I think this should also just be Mackenzie, not MacKenzie.

**Review by mattchrlw — COMMENTED** [review]
See above

File: committee_history.md (hunk 1 above)
Comment (nicklambourne) [reply committee_history.md:121]: Agreed 👍  Fixed

**Review by JamesDearlove — APPROVED** [review]
Hey, not my fault the last PR was handwaved through :P

**Pull request merged**